cmd/kubeadm/app/cmd/flags/authentication_mode.go

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
ok, let's remove this flag, it should only be exposed via the API
let's focus on enabling rbac instead